src/main/java/org/pac4j/dropwizard/Pac4jFactory.java

```diff
@@ -329,44 +369,72 @@ public void setSessionEnabled(boolean sessionEnabled) {
     }
 
     public Config build() {
-        Clients client = new Clients();
-        JaxRsConfig config = new JaxRsConfig();
+        // either the whole Config is built from a ConfigFactory class, or we initialize a default JaxRsConfig
+        final JaxRsConfig config;
+        if (configClass != null && !configProperties.isEmpty()) {
+            config = (JaxRsConfig) ConfigBuilder.build(configClass, configProperties);
```

Comment:
No, it might be of another type. In that case, a fairly obvious `ClassCastException` is thrown, but I can explicitly do the check and throw a `TechnicalException` and an even more explicit message...